components/landlord-setup/listing-confirmation.tsx

```
@@ -0,0 +1,427 @@
+							</>
+						)}
+					</label>
+				</div>
+
+				{/* Photo preview */}
+				<div className="grid grid-cols-2 md:grid-cols-3 gap-4">
+					{photos.map((photo, index) => (
+						<div key={index} className="relative group">
+							<img
+								src={photo}
+								alt={`Property photo ${index + 1}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)